doc/integrator/authentication.rst

```diff
@@ -27,6 +27,19 @@ table. If a c2cgeoportal application should work with another user information
 source, like LDAP, a custom *client validation* mechanism can be set up.
 Our knowledge base has an example of how this can be achieved.
 
+Basic auth
+~~~~~~~~~~
+
+To be able acces to the OGC services from your desktop GIS you should anable the basic authentication
+by setting ``BASICAUTH`` to ``True`` in the ``.env.sample`` file.
+
+To force the application to ask for a password you should have the attribute ``authentication_required``
+in your querry string.
+
+.. note::
+
+   For security reason the basic auth and the two factors authentication shouldn't be enahabled toogether.
```

Comment:
For security reasons, basic authentication and two factor authentication should not be enabled together.